DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/README.md

```diff
@@ -89,6 +90,7 @@ The following block diagram shows an overview of a typical oneAPI FPGA BSP hardw
 | `board_test.hpp`   | Contains the definitions for all the individual tests in the sample.
 | `host_speed.hpp`   | Header for host speed test. Contains definition of functions used in host speed test.
 | `helper.hpp`       | Contains constants (for example, binary name) used throughout the code as well as definition of functions that print help and measure execution time.
+| `usm_speed.hpp`    | Header for the USM bandwidth test. Contains definitions of functions used in the USM bandwidth test.
 
 ### Compiler Flags Used
 

```

Comment:
This comment is for line 137and 176:

does it make sense to target this design to IP authoring? if not, we can probably remove the note about choosing an FPGA device family or part number.